Test failure:
```
[ERROR] Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.009 s <<< FAILURE! - in com.yahoo.search.query.SoftTimeoutTestCase
[ERROR] testLimits(com.yahoo.search.query.SoftTimeoutTestCase)  Time elapsed: 0.006 s  <<< FAILURE!
org.junit.ComparisonFailure: expected:<...the range [0.0, 1.0][. It is] -0.1> but was:<...the range [0.0, 1.0][, got] -0.1>
	at com.yahoo.search.query.SoftTimeoutTestCase.verifyException(SoftTimeoutTestCase.java:40)
	at com.yahoo.search.query.SoftTimeoutTestCase.testLimits(SoftTimeoutTestCase.java:45)

```

--- a/container-search/src/test/java/com/yahoo/search/query/SoftTimeoutTestCase.java
+++ b/container-search/src/test/java/com/yahoo/search/query/SoftTimeoutTestCase.java
@@ -37,7 +37,7 @@ public class SoftTimeoutTestCase {
         } catch (QueryException e) {
             assertEquals("Invalid request parameter", e.getMessage());
             assertEquals("Could not set 'ranking.softtimeout." + key + "' to '" + value +"'", e.getCause().getMessage());
-            assertEquals(key + " must be in the range [0.0, 1.0]. It is " + value, e.getCause().getCause().getMessage());
+            assertEquals(key + " must be in the range [0.0, 1.0], got " + value, e.getCause().getCause().getMessage());
         }
     }
     @Test

